src/test/scala/codacy/metrics/PDependSpec.scala

```diff
@@ -0,0 +1,39 @@
+package codacy.metrics
+
+import codacy.docker.api.Source
+import codacy.docker.api.metrics.{FileMetrics, LineComplexity}
+import com.codacy.api.dtos.Languages
+import org.specs2.mutable.Specification
+
+class PDependSpec extends Specification {
```

Comment:
Just to be sure, add a test to check the Failure when a different language is provided.